berkeley-mobile/Fitness/Fitness+Controllers/GymClassesController.swift

```diff
@@ -58,8 +58,27 @@ extension GymClassesController: UITableViewDataSource {
             let gymClass = vc.todayClasses[indexPath.row]
             cell.cellConfigure(entry: gymClass, type: gymClass.type, color: gymClass.color)
             cell.eventTime.text = gymClass.description(components: [.startTime, .duration, .location])
+            cell.eventCategory.text = gymClass.type
+            cell.eventCategory.backgroundColor = gymClass.color
+            cell.eventCategory.isHidden = gymClass.type == nil
+            cell.eventTaggingColor.backgroundColor = gymClass.color
+            
+            if let url = gymClass.website_link, gymClass.location == "Zoom" {
+                let tapGesture = EventTapGestureRecognizer(target: self, action:
+                                                            #selector(GymClassesController.zoomTapped(gesture:)))
+                tapGesture.eventUrl = URL(string: url)
+                
+                cell.cellSetImage(image: UIImage(named: "Zoom Logo")!,
+                                  tapGesture: tapGesture)
+            }
         }
         return cell
     }
     
+    @objc func zoomTapped(gesture: EventTapGestureRecognizer) {
+        if (gesture.view as? UIImageView) != nil {
+            vc.presentAlertLinkUrl(title: "Are you sure you want to open Zoom?", message: "Berkeley Mobile wants to open an online fitness class in Zoom", options: "Cancel", "Yes", website_url: gesture.eventUrl!)
```

Comment:
Could you do a quick check if `gesture.eventUrl` is non-nil (i.e. if the string `gymClass.website_link` is not formatted properly as an URL, I think this will still be nil). Should just be able to add a comma to the above if condition and do a quick `let url = gesture.eventUrl`.